packages/backend/src/services/executionContractTrustGraph/tenancyOwnershipHttpService.ts

```
@@ -0,0 +1,171 @@
+
+            if (!response.ok) {
+                throw new Error(
+                    `tenancy_ownership_http_status_${response.status}`
+                );
+            }
+
+            const payload = (await response.json()) as unknown;
+            return validateResponseShape(payload);
+        } catch (error: unknown) {
+            if (timeoutTriggered) {
+                throw new Error('ownership_validator_timeout');
```

ESLint

[error] There is no `cause` attached to the symptom error being thrown.

(preserve-caught-error)